changelog.md

```diff
@@ -2,7 +2,52 @@
 
 ## 2.8.0~rc1
 
+### Web applications
+* Updated strings based on translator feedback (#7057)
+* Improved redwood stream performance and testing (#7070)
+* Dependency changes:
+  * openssl rust crate from 0.10.57 to 0.10.60 (#7083)
+  * cryptography from 41.0.3 to 41.0.7 (#7086)
+  * rustix rust crate from 0.38.18 to 0.38.21 (#7114)
+
+### Operations
+* Updated copyright strings to reference 2024 (#7099)
+* Removed deprecated mitigation for CVE-2019-3462 (#7053)
+* Improved logic for installing admin tool apt dependencies in Tails (#7088)
+* Added support for Tails 6 to admin tools (#7116)
```

Comment:
Do you want to list the various dependency updates that went into this?

Reply:
We could - I've defaulted to listing updates with a specific PR but an exhaustive list wouldn't be hard to generate.